ygen/protoelements.go

```diff
@@ -327,6 +395,11 @@ func (s *ProtoLangMapper) yangTypeToProtoScalarType(args resolveTypeArgs, pargs
 	}
 }
 
+type unionSubtypeInfo struct {
```

Comment:
Can we add a comment here or move the type definition into the function itself since it seems to only be used in one place.

Reply:
The diff was bad. Please take a look at the diff now.